GetSubscriptionsFeed/__tests__/handler.test.ts

```diff
@@ -0,0 +1,331 @@
+/* tslint:disable:no-any */
+/* tslint:disable:no-duplicate-string */
+/* tslint:disable:no-big-function */
+/* tslint:disable: no-identical-functions */
+
+import { TableService } from "azure-storage";
+import * as dateFmt from "date-fns";
+import { ServiceId } from "io-functions-commons/dist/generated/definitions/ServiceId";
+import { FiscalCodeHash } from "../../generated/definitions/FiscalCodeHash";
+import { GetSubscriptionsFeedHandler } from "../handler";
+
+const today = new Date();
+const tomorrow = new Date(today);
+tomorrow.setDate(tomorrow.getDate() + 1);
+
+const yesterday = new Date(today);
+yesterday.setDate(yesterday.getDate() - 1);
+const aServiceId = "MyServiceId" as ServiceId;
+
+const yesterdayUTC = dateFmt.format(yesterday, "YYYY-MM-DD");
```

Comment:
<3

Reply:
I've noticed this: https://date-fns.org/v1.28.5/docs/startOfYesterday maybe can help

Reply:
Done 👍  [047a2f4](https://github.com/pagopa/io-functions-services/pull/86/commits/047a2f4aa4010e7e319766b0923bca4ca17af2d4)